File: 1. Beginner/1008/1008.dart
import 'dart:io';

void main() {
  int n = int.parse(stdin.readLineSync()!)!;
  int m = int.parse(stdin.readLineSync()!)!;
  double s = double.parse(stdin.readLineSync()!)!;
  
  print("NUMBER = $n\nSALARY = U\$ ${(m! * s!).toStringAsFixed(2)}");
}

File: 1. Beginner/1008/1008.js
var inputs = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');

var n = parseInt(inputs.shift());
var m = parseInt(inputs.shift());
var s = parseFloat(inputs.shift());
   
console.log(`NUMBER = ${n}\nSALARY = U$ ${(m * s).toFixed(2)}`);